src/util/browser.js

```diff
@@ -19,6 +19,8 @@ const cancel = window.cancelAnimationFrame ||
 
 let linkEl;
 
+let _reducedMotionQuery: MediaQueryList;
```

Comment:
Nit: maybe remove the dangling `_` in the name for consistency? This will be private anyway since it's not exposed.